src/VocabGeneration.test.js

```diff
@@ -231,12 +231,32 @@ describe("Suite for generating common vocabularies (marked as [skip] to prevent
   // it("tests a single custom vocab", async () => {
   it.skip("tests a single custom vocab", async () => {
     await generateVocabArtifact({
+      inputResources: ["http://www.w3.org/2006/03/test-description#"],
+      nameAndPrefixOverride: "td",
+      descriptionFallback: "Test description",
```

Comment:
There is nothing specific about this vocabulary in the test description. Why not have a dedicated test we want to have for non-regression on this vocab?

Reply:
I don't follow you. This is the 'Test Description' vocabulary, and it doesn't explicitly provide a description of itself (i.e., it has no `rdfs:comment` or `dctems:title` triples), so we need to provide one here.

And this entire function isn't a 'test' per se at all, it's just the quickest, easiest way to run the AG against individual vocabs for investigation (so no need for a real test, and no need for nn-regression of the specific vocab).

Reply:
I don't follow you. This is the 'Test Description' vocabulary, and it doesn't explicitly provide a description of itself (i.e., it has no `rdfs:comment` or `dctems:title` triples), so we need to provide one here.

And this entire function isn't a 'test' per se at all, it's just the quickest, easiest way to run the AG against individual vocabs for investigation (so no need for a real test, and no need for non-regression of the specific vocab).

Reply:
> And this entire function isn't a 'test' per se at all, it's just the quickest, easiest way to run the AG against individual vocabs for investigation (so no need for a 'real' test, and no need for non-regression of the specific vocab).

Then this shouldn't be versioned as part of the code, it should just run on an individual developer's machine. What's the value of this being tracked on git?